testing/opae-cxx/test_object_cxx_core.cpp

```diff
@@ -117,9 +132,17 @@ TEST_P(sysobject_cxx_p, handle_object) {
  * Then no exceptions are thrown
  */
 TEST_P(sysobject_cxx_p, handle_object_write) {
-  auto obj = sysobject::get(handle_, "errors/errors");
+  std::string path = "iperf/fabric/freeze";;
+
+  if (platform_.devices[0].device_id == 0x09c4 || 
```

Comment:
Hmm.. do we need an is_dcp() function?

Reply:
Probably should implement something like this in fpga_hw.cpp at some point.